docs/book/v3/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -93,6 +94,7 @@ return [
     // …
 ];
 </code></pre>
+```
```

Comment:
This change leads to a doubling of the `<pre>` tags.

Use `<!-- markdownlint-disable-next-line no-inline-html -->` instead.

https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md033.md

Reply:
Fixed as per your recommendation